spec/controllers/availability_controller_spec.rb

```diff
@@ -41,19 +41,22 @@
         "2282456310006421" => {
           "more_items" => false,
           "location" => "LAW-LAWRR",
-          # "copy_number" => 1, TODO => Implement
-          "item_id" => "2382456270006421"
+          "copy_number" => "",
+          "item_id" => "2382456270006421",
           # "on_reserve": "N", TODO: Implement
           # "patron_group_charged": null, TODO: Implement
-          # "status": "On-Site", TODO: Implement
-          # "label": "Marquand Library", TODO: Implement
-          # "status_label": "On-site access" TODO: Implement
+          "status" => "On-Site", # TODO: Implement
```

Comment:
Since you have included the status can you remove the TODO: Implement from line 48 ?

Reply:
@christinach Ready for another look.